AK/Error.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Error.h>
+
+#ifdef KERNEL
+#    include <AK/Format.h>
+#endif
+
+namespace AK {
+
+Error from_string_view_or_kernel_errno(StringView string_literal, [[maybe_unused]] int code)
+{
+#ifdef KERNEL
+    dmesgln("{}", string_literal);
```

Comment:
I don't think this is a useful thing to have, the name heavily implies that the string will be part of the created error instance, and not just printed out.
Let's just do that explicitly and then remove `from_string_view()` from the kernel build.

Reply:
Updates from discord:
- Limit uses of this function to AK/LibEDID/other userland-kernel shared libs only
- Rename to `from_string_view_or_print_error_and_return_errno` for clarity

Reply:
Done!